packages/models/src/models/CalendarEvent.ts

```diff
@@ -59,10 +69,12 @@ export class CalendarEventRaw extends BaseRaw<ICalendarEvent> implements ICalend
 					...(subject !== undefined ? { subject } : {}),
 					...(description !== undefined ? { description } : {}),
 					...(startTime ? { startTime } : {}),
+					...(endTime ? { endTime } : {}),
 					...(meetingUrl !== undefined ? { meetingUrl } : {}),
 					...(reminderMinutesBeforeStart ? { reminderMinutesBeforeStart } : {}),
 					...(reminderTime ? { reminderTime } : {}),
 					...(previousStatus ? { previousStatus } : {}),
+					...(typeof busy === 'boolean' && { busy }),
```

Comment:
same change two different ways to use spread
```suggestion
					...(endTime && { endTime }),
					...(meetingUrl !== undefined ? { meetingUrl } : {}),
					...(reminderMinutesBeforeStart ? { reminderMinutesBeforeStart } : {}),
					...(reminderTime ? { reminderTime } : {}),
					...(previousStatus ? { previousStatus } : {}),
					...(typeof busy === 'boolean' && { busy }),
```